### ISSUE TITLE
Accessibility issues on my website 

### ISSUE THREAD

**Issue by alamas1** [pojome/elementor#4984]
Hi,

Can you please look into fixing the below issues? I reached out to Codeable and they said I should report this to you to fix it and improve accessibility within the plugin.

Thanks,
Andrea

Source URL: https://onetoronto.com/
Source Title: One Toronto Gaming | Entertainment Re-imagined
Accessibility Review (Guidelines: WCAG 2.0 (Level AA))
Report on known problems (14 found):

**ISSUE #1**
3.3 Input Assistance: Help users avoid and correct mistakes.
Success Criteria 3.3.2 Labels or Instructions (A)
Check 188: Label text is empty.
**Repair: Add text to the label element**.
Line 997, Column 3:
<input type="search" name="s" autocomplete="off" placeholder="Search" />

**ISSUE #2**
1.4 Distinguishable: Make it easier for users to see and hear content including
separating foreground from background.
Success Criteria 1.4.4 Resize text (AA)
Check 117: i (italic) element used.
**Repair: Replace your i elements with em or strong.**
Line 263, Column 4:
<i class="eicon" aria-hidden="true"></i>
Line 294, Column 6:
<i class="fa fa-linkedin"></i>
Line 298, Column 6:
<i class="fa fa-envelope"></i>
Line 511, Column 5:
<i class="fa fa-diamond" aria-hidden="true"></i>
Line 614, Column 8:
<i class="eicon-chevron-left"></i>
Line 617, Column 8:
<i class="eicon-chevron-right"></i>
Line 788, Column 6:
<i class="fa fa-envelope"></i>
Line 792, Column 6:
<i class="fa fa-linkedin"></i>
Line 816, Column 8:
<i class="fa fa-phone" aria-hidden="true"></i>
Line 832, Column 8:
<i class="fa fa-phone" aria-hidden="true"></i>
Line 848, Column 8:
<i class="fa fa-phone" aria-hidden="true"></i>
Line 990, Column 3:
<i class="icon icon-close"></i>
Line 999, Column 4:
<i class="icon icon-magnifier"></i>

**Comment by bainternet** [comment]
@alamas1 

Thanks for reporting, but this is not a support channel and unless you have a specific defined issue, we can't really help.

thanks again.

**Comment by alamas1** [comment]
Hey,

The support team sent me here. I did some research over the weekend and it looks like the I elements for the icons are not accessible. I'm trying to fix them by creating my own icons with span tags and/or images, but the Elementor text widget deletes the span tags every time I go back to edit them.

This is an issue with the icons and how they are coded. Can you recommend a solution? 

Thanks,
Andrea
